modules/stats.py

```diff
@@ -439,6 +444,7 @@ def report_stats(removed_titles: Removes, config: Config) -> None:
     eprint(f'{total_titles}{Font.b}')
     eprint('-' * len(total_titles))
     eprint(f'=  Final title count: {config.stats.final_count:,}{Font.be}')
+    eprint(f'=  Final size: {config.stats.final_size:,}{Font.be}')

```

Comment:
From a usability perspective, this gives a number with no units, leading to confusion:

```
Final size: 2,000
```

2,000 what? The user doesn't know.

To make this user-friendly, it should scale through the common tiers and label accordingly..

1,023 B
1,023 KiB
1,023 MiB
1,023 GiB
1,023 TiB